src/aggregate/aggregate_request.c

```diff
@@ -1196,20 +1206,14 @@ int buildOutputPipeline(AREQ *req, QueryError *status) {
 int AREQ_BuildPipeline(AREQ *req, int options, QueryError *status) {
   if (!(options & AREQ_BUILDPIPELINE_NO_ROOT)) {
     buildImplicitPipeline(req, status);
+    if (status->code != QUERY_OK) {
```

Comment:
How come we did not do it before? Was it a bug?

Reply:
Before it seems like the two result processors that were created in this function could not fail, so there was no reason to check the error code